src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -368,3 +384,24 @@ def get_request_preview_queries(
             status_code=HTTP_400_BAD_REQUEST,
             detail=f"Dry run failed",
         )
+
+
+@router.post(
+    PRIVACY_REQUEST_RESUME,
+    status_code=200,
+)
+def resume_privacy_request(
+    privacy_request_id: str,
+    *,
+    db: Session = Depends(deps.get_db),
+    cache: FidesopsRedis = Depends(deps.get_cache),
+    webhook: PolicyPreWebhook = Security(
+        verify_callback_oauth, scopes=[scopes.PRIVACY_REQUEST_CALLBACK_RESUME]
+    ),
+    webhook_callback: PrivacyRequestResumeFormat,
+) -> None:
+    """Resume running a privacy request after it was paused by a Pre-Execution webhook"""
+    privacy_request = get_privacy_request_or_error(db, privacy_request_id)
+    privacy_request.cache_identity(webhook_callback.derived_identities)
+
+    # TODO resume running privacy request from specific webhook
```

Comment:
This will be connected in follow-up #101, currently no webhooks are actually run.